src/app/core/services/scheduling.service.ts

```diff
@@ -118,7 +122,7 @@ export class SchedulingService {
             if (tasks[i]) {
               if (
                 tasks[i].reportedCompletion === false &&
-                tasks[i].timestamp < now &&
+                tasks[i].timestamp + this.ONE_DAY < now &&
```

Comment:
Can this be configurable in the defaultConfig too ? Because if we increase the completion window (like 3 days for PHQ8 as discussed in the steering committee meeting) then this will need to be updated. Also would be good to add a comment to document what is this and why this is used for future reference.

Reply:
Fixed in another pull request.

Reply:
ok